### ISSUE TITLE
Arrange a scoping session to kick off 'show / hide' password variant

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-frontend#4090]
## What
Arrange a kick off session to scope the work to add a 'show / hide' password variant.

## Why
So we understand:
- why we're doing this
- who needs to be involved
- what we're going to do (understanding prerequisites, major delivery milestones, dependencies)
- risks
- timescales for delivery

## Who needs to work on this
Developer, Designer, Accessibility Specialist

## Who needs to review this
Product Manager, Delivery Manager, Content Designer

## Done when
- [x] Assigned an epic lead
- [x] Planned a kick off session
- [x] Run a kick off session
- [x] Documented decisions
- [x] Updated [epic card](https://github.com/alphagov/govuk-frontend/issues/4063) with plan of action

**Comment by Ciandelle** [comment]
Beeps assigned as epic lead during this session, however due to a lack of time we will need to run a second session. I have already created a github [issue](https://github.com/alphagov/govuk-frontend/issues/4153) for this, therefore this issue can now be closed.
